Dockerfile

```diff
@@ -1,16 +1,9 @@
 # syntax=docker.io/docker/dockerfile:1.13-labs
 # Pelican Production Dockerfile
 
-
-# For those who want to build this Dockerfile themselves, uncomment lines 6-12 and replace "localhost:5000/base-php:$TARGETARCH" on lines 17 and 67 with "base".
-
-# FROM --platform=$TARGETOS/$TARGETARCH php:8.4-fpm-alpine as base
-
-# ADD --chmod=0755 https://github.com/mlocati/docker-php-extension-installer/releases/latest/download/install-php-extensions /usr/local/bin/
-
-# RUN install-php-extensions bcmath gd intl zip opcache pcntl posix pdo_mysql pdo_pgsql
-
-# RUN rm /usr/local/bin/install-php-extensions
+##
+#  If you want to build this locally you want to run `docker build -f Dockerfile.dev`
+##
```

Comment:
I don't think duplicating the entire Dockerfile is a good idea. I'm going to review this in more detail later and come up with a suggestion for this instead.

Reply:
That's fine for a DEV one